features/support/hooks.rb

```
@@ -19,6 +19,16 @@
     Howitzer::Log.info 'IE reset session'
     Capybara.current_session.execute_script("void(document.execCommand('ClearAuthenticationCache', false));")
   end
+
+  test_teardown = Howitzer::Cache.extract(:teardown)
+  begin
+    @category.destroy if test_teardown.key? :category
```

Das ist nicht praktisch! ;)

```ruby
test_teardown.keys.each do |key|
  instance_variable_get("@#{key}")&.destroy
end
```
Suppressing errors is not good idea.